modules/openapi-generator/src/main/resources/ruby-client/api_client.mustache

```diff
@@ -91,6 +91,12 @@ module {{moduleName}}
       end
       {{/isFaraday}}
 
+      # return byte stream for Binary return type
+      if return_type == 'Binary'
+        encoding = body.encoding
+        return @stream.join.force_encoding(encoding)
```

Comment:
I think @stream is only available in `faraday` library:
```
grep -R "@stream" modules/openapi-generator/src/main/resources/ruby-client/*
modules/openapi-generator/src/main/resources/ruby-client/api_client.mustache:        @tempfile.write(@stream.join.force_encoding(encoding))
modules/openapi-generator/src/main/resources/ruby-client/api_client.mustache:        return @stream.join.force_encoding(encoding)
modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache:      @stream = []
modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache:        @stream << chunk
```

Reply:
FYI. I've filed https://github.com/OpenAPITools/openapi-generator/pull/11974 to fix it.